### ISSUE TITLE
Export a separate CSV file with training sessions details

### ISSUE THREAD

**Issue by mkduer** [mkduer/reinforcement-learning-snake-game#56]
- [x] grid size
- hyperparameters
  - [x] learning rate
  - [x] discount factor
  - [x] epsilon (pending implementation)
- [x] reward/penalty values
- [x] total training episodes
